stl/inc/xstring

```diff
@@ -698,7 +698,7 @@ constexpr size_t _Traits_rfind_ch(_In_reads_(_Hay_size) const _Traits_ptr_t<_Tra
 template <class _Elem, bool = _Is_character<_Elem>::value>
 class _String_bitmap { // _String_bitmap for character types
 public:
-    constexpr bool _Mark(const _Elem* _First, const _Elem* const _Last) {
+    constexpr bool _Mark(const _Elem* _First, const _Elem* const _Last) noexcept {

```

Comment:
Honestly, these four added `noexcept`s have nothing to do with LWG-3518 or character trait operations, I simply noticed in passing that they are do not emit exceptions when operating on character-like types.